src/Message.php

```diff
@@ -16,7 +16,7 @@
     public function __construct($payload, $timestamp, $encoding)
     {
         $this->payload = $payload;
-        $this->timestamp = new \DateTimeImmutable($timestamp);
+        $this->timestamp = new \DateTimeImmutable("@$timestamp");
```

Comment:
I think we could use this \DateTime::ISO8601 in UTC as default usage, better then unix timestamp